src/lib/components/ProductForm.svelte

```diff
@@ -168,6 +170,11 @@
 				// rerun all `load` functions, following the successful update
 				await invalidateAll();
 			}
+			if (result.type === 'error') {
+				errorMessage =
+					result.error.message === 'Internal Error' ? 'Duplicate alias' : 'Error on submission';
```

Comment:
The error message should be forwarded as is, it's up to the backend to  produce a readable message.

There is a `isUniqueConstraintError` helper, you can do a try/catch with in the catch something like

```ts
if (isUniqueContraintError(err)) {
  throw error(400, 'A product with the same alias already exists');
} else {
	throw err;
}
```

Maybe you can even check the key with the unique constraint from the contents of `err` (but not required)